qemu/tests/x86_cpu_model.py

```diff
@@ -75,6 +75,21 @@ def run(test, params, env):
         test.fail("Guest cpu model is not right")
 
     if params["os_type"] == "linux":
+        check_cmd = params.get("check_cmd")
+        if check_cmd:
+            vul_host = process.getoutput(params.get("vulnerabilities")).split()
+            vul_guest =  session.cmd_output(params.get("vulnerabilities")).split()
+            vulnerabilities = list(set(vul_host).intersection(set(vul_guest)))
+            for item in vulnerabilities:
+                h_out = re.search("Vulnerable|Mitigation|Not affected",
+                                  process.getoutput(check_cmd % item))[0]
+                g_out = re.search("Vulnerable|Mitigation|Not affected",
+                                  session.cmd_output(check_cmd % item))[0]
+                if h_out != g_out:
+                    test.fail("Guest is not equal to Host with %s" % item)
+                if item in ['itlb_multihit', 'mds', 'tsx_async'] \
+                        and g_out != 'Not affected':
```

Comment:
You may need consider if new cases in the future will check cpu vulnerabilities too and if current code suits.

Reply:
@huangyum thank and update, please help to review.